client/src/main/scala/grasshopper/client/addresspoints/AddressPointsClient.scala

```diff
@@ -0,0 +1,51 @@
+package grasshopper.client.addresspoints
+
+import java.io.IOException
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import com.typesafe.config.ConfigFactory
+import feature.Feature
+import io.geojson.FeatureJsonProtocol._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import grasshopper.client.ServiceClient
+import grasshopper.client.addresspoints.model.AddressPointsStatus
+import grasshopper.client.addresspoints.protocol.AddressPointsJsonProtocol
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.Properties
+
+object AddressPointsClient extends ServiceClient with AddressPointsJsonProtocol {
+  override val config = ConfigFactory.load()
+
+  lazy val host = Properties.envOrElse("GRASSHOPPER_ADDRESSPOINTS_HOST", config.getString("grasshopper.client.addresspoints.host"))
+  lazy val port = Properties.envOrElse("GRASSHOPPER_ADDRESSPOINTS_PORT", config.getString("grasshopper.client.addresspoints.port"))
+
+  def status(): Future[Either[String, AddressPointsStatus]] = {
+    implicit val ec: ExecutionContext = system.dispatcher
+    sendGetRequest(host, port.toInt, "/status").flatMap { response =>
+      response.status match {
+        case OK => Unmarshal(response.entity).to[AddressPointsStatus].map(Right(_))
+        case BadRequest => Future.successful(Left("Bad Request"))
+        case _ => Unmarshal(response.entity).to[String].flatMap { entity =>
+          val error = s"Request failed with status code ${response.status} and entity ${entity}}"
+          Future.failed(new IOException(error))
```

Comment:
Why `IOException`?  If you get a response then the I/O was successful.  It seems like it should be an unchecked exception...but then again I don't really understand what the end result of wrapping the an exception in `Future.failed()`, so maybe not.